feature(claude-code): testing AI writing assistant (#2960)
kestra-io/docs: CLAUDE.md

# CLAUDE.md

This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.

# Kestra documentation

## Working relationship
- You can push back on ideas-this can lead to better documentation. Cite sources and explain your reasoning when you do so
- ALWAYS ask for clarification rather than making assumptions
- NEVER lie, guess, or make up information

> - If you are making an inferrance, stop and ask me for confirmation or say that you need more information

Reviewer: ```suggestion
- Instead of making assumptions, stop and ask me for confirmation or say that you need more information
```
